bats/sql-delimiter.expect

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/expect
```

Comment:
Probably want this to include at least one test of multiple statements like a `BEGIN .. END` block. The only place this is supported in the grammar right now is when creating a trigger.